### PR TITLE
Analytics dashboard api

### PR DESCRIPTION
**Pull request by SebaKas21**
Hi @dawiddworak88 

I was thinking to create two endopoints. The first is for displaying the top 10 products and the second is for the chart.

Check this out.

### CODE REVIEW

**Comment by SebaKas21** [comment]
Hi @dawiddworak88 

You can make review :D

**Comment by dawiddworak88** [comment]
Hi @SebaKas21 , could you please merge develop to the branch and add open telemetry?

File: be/src/Project/Services/Analytics/Analytics.Api/v1/Controllers/SalesAnalyticsController.cs [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+                    ClientName = x.ClientName,
+                    Email = x.Email,
+                    ProductId = x.ProductId,
+                    ProductName = x.ProductName,
+                    ProductSku = x.ProductSku,
+                    Ean = x.Ean,
+                    IsOutlet = x.IsOutlet,
+                    IsStock = x.IsStock,
+                    Quantity = x.Quantity,
+                    Country = x.Country,
```
Comment (dawiddworak88) [inline be/src/Project/Services/Analytics/Analytics.Api/v1/Controllers/SalesAnalyticsController.cs]: **suggestion:** let's merge the develop branch to this one. Let's use country id and introduce the dimension country with translation for name.

**Review by dawiddworak88 — CHANGES_REQUESTED** [review]

**Review by dawiddworak88 — APPROVED** [review]
Looks good, let's develop on the top of it 🌞

**Pull request merged**